core/components/molecules/dialog/dialog.md

```diff
@@ -82,3 +87,38 @@ class DialogContainer extends React.Component {
   }
 }
 ```
+
+## Usage
+
+As a general rule, try to avoid using dialogs: They are disruptive, invasive, and interrupt the user's flow.
+
+Use dialogs only to allow the user to maintain the context of a particular task.
+
+Good use of dialogs are:
+
+- **Transactional:** Dialogs that allow user to validate irreversible actions.
+
+Try to avoid dialogs for:
+
+- **Passive:** Dialogs that can be replaced for notifications.
+- **Input:** Dialogs that include forms.
+
+When you see the need to create a dialog, ask yourself if this interaction must completly stop the user's flow. If the answer is no, then consider other options to display the information.
+
+## Examples
+
+- Default passive modal (missing code snippet)
```

Comment:
Is this pending or should we remove this from the docs?

Reply:
This is the default role of the dialog (general content). I will add the missing snippet.

Reply:
Checking on the props for `role` there are only two options: form and destructive. I'm guessing we are missing a `general` or `content` (even as a default) prop for the use cases where none of these two apply. Still, the user may not understand why this role needs to be set.

Reply:
Added `default` role as default.